src/Libraries/CoreNodes/CoreNodes.csproj

```diff
@@ -20,10 +20,6 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
     <Reference Include="Microsoft.CSharp" />
-    <PackageReference Include="ncalc" Version="1.3.8">
-      <!--Exclude copying the runtime dlls because we need to keep binary compatibility with Revit for now -->
-      <ExcludeAssets>runtime</ExcludeAssets>
-    </PackageReference>
```

Comment:
Do we still have NCalc around in other projects ? If not then we may need to copy it for net48 for compatibility

Reply:
Good point, will do.